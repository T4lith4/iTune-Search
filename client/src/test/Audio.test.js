import React from "react";
import Audio from "../components/audio-books";
import renderer from "react-test-renderer";

test("renders correctly", () => {
  const tree = renderer.create(<Audio />).toJSON();
  expect(tree).toMatchSnapshot();
});