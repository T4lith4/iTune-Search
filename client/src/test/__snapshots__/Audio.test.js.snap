// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container"
>
  <h4
    className="welcome-text"
  >
    Add audiobooks to your favorite list by clicking on the "Add as Favorite" button 
    <br />
    <br />
    View them at the bottom of the page
  </h4>
  <br />
  <label
    className="search-label"
    htmlFor="search-input"
  >
    <input
      id="search-input"
      name="query"
      onChange={[Function]}
      placeholder="Search audiobook author"
      type="text"
      value=""
    />
    <i
      aria-hidden="true"
      className="fa fa-search search-icon"
    />
  </label>
  <img
    alt="loader"
    className="search-loading hide"
    src="loader.gif"
  />
  <div>
    <table
      className="table table-bordered"
    >
      <tbody>
        <tr
          className="row-container"
        />
      </tbody>
    </table>
  </div>
  <br />
  <div
    className="nav-link-container"
  >
    <a
      className="nav-link hide 
          "
      href="/"
      onClick={[Function]}
    >
      ←
    </a>
    <a
      className="nav-link hide
          "
      href="/"
      onClick={[Function]}
    >
      →
    </a>
  </div>
</div>
`;
